### PR TITLE
feat: added Config classes on BE & FE

### PR DESCRIPTION
**Pull request by GaelFerrand**

### CODE REVIEW

File: ammo-fe/src/util/Config.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+}
+
+const buildConfig = (): IConfig => {
+    return {
+        beUrl: process.env.REACT_APP_BE_URL,
+        port: parseInt(process.env.PORT, 10),
+    };
+};
+
+export default buildConfig();
```
Comment (GaelFerrand) [inline ammo-fe/src/util/Config.ts:11]: @Spartakyste  I followed your advice and tried to use functions and not classes when not needed! :)

File: ammo-be/src/util/Config.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/no-unresolved */
+// eslint-disable-next-line import/extensions
+import config from 'config/config.json';
+import packageJSON from 'package.json';
+import PackageJSON from 'types/PackageJSON';
```
Comment (chandelieraxel) [inline ammo-be/src/util/Config.ts:3]: I do have troubles with the naming, why adding JSON ?

File: ammo-be/src/util/Config.ts [hunk 3]
```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/no-unresolved */
```
Comment (chandelieraxel) [inline ammo-be/src/util/Config.ts]: I think we should remove both lines and investigate the linter errors

File: ammo-be/src/util/Config.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import ConfigJSON from 'types/ConfigJSON';
+
+// Final config is a merge of package.json & config.json properties,
+// with some added props
+interface IConfig extends PackageJSON, ConfigJSON {
+    url: string;
+}
+
+const buildConfig = (): IConfig => {
+    const castPackage = (packageJSON as unknown) as PackageJSON;
```
Comment (chandelieraxel) [inline ammo-be/src/util/Config.ts]: Is the as unknown necessary ? Can't we do some TS magic ?

File: ammo-fe/src/util/Config.ts (hunk 1 above)
Comment (chandelieraxel) [reply ammo-fe/src/util/Config.ts:11]: @GaelFerrand Could be simplified ! 
const buildConfig = (): IConfig => ({
         beUrl: process.env.REACT_APP_BE_URL,
         port: parseInt(process.env.PORT, 10),
     });;

File: ammo-be/src/util/Config.ts (hunk 2 above)
Comment (GaelFerrand) [reply ammo-be/src/util/Config.ts:3]: 'package' is a reserved keyword in JS, packageJSON seemed fair to me as it's importing the package.json

**Pull request merged**